helix-core/src/main/java/org/apache/helix/HelixPropertyFactory.java

```diff
@@ -80,20 +80,23 @@ public HelixManagerProperty getHelixManagerProperty(String zkAddress, String clu
    * @param clusterName
    * @return
    */
-  public static CloudConfig getCloudConfig(String zkAddress, String clusterName) {
+  public static CloudConfig getCloudConfig(String zkAddress, String clusterName,
+      RealmAwareZkClient.RealmAwareZkConnectionConfig realmAwareZkConnectionConfig) {
```

Comment:
This is a public method, could we maintain the original interface while adding a new one?

Reply:
Good point. Updated.